openehr-rm/src/main/java/com/nedap/archie/rminfo/UpdatedValueHandler.java

```diff
@@ -145,8 +159,8 @@ private static ArchetypeTerm getTermDefinition(OperationalTemplate template, Arc
                 if(child instanceof CTerminologyCode) {
                     String value = ((CTerminologyCode) child).getConstraint().get(0);
                     if(value.startsWith("ac")) {
-                        codedText.getDefiningCode().setTerminologyId(new TerminologyId(value));
-                        result.put(path + "/defining_code/terminology_id/value", value);
+                        codedText.getDefiningCode().setTerminologyId(new TerminologyId("local"));
+                        result.put(path + "/defining_code/terminology_id/value", "local");
```

Comment:
Is the `setTerminologyFromArchetype` method still needed with your change setting the terminology id in `setDvCodedTextValue` above?

Reply:
Good catch. Removed.

Reply:
This method is now unused, can you remove it?